src/api/Nhs.Appointments.Persistance/SiteStore.cs

```diff
@@ -50,6 +50,37 @@ public async Task<OperationResult> UpdateSiteAttributes(string siteId, string sc
         return new OperationResult(true);
     }
     
+    public async Task<OperationResult> UpdateSiteDetails(string siteId, string name, string address, string phoneNumber, string latitude, string longitude)
+    {
+        //TODO move away parsing?
+        var latitideParseResult = decimal.TryParse(latitude, out var latitudeDecimal);
+        var longitudeParseResult = decimal.TryParse(longitude, out var longitudeDecimal);
+
+        if (!latitideParseResult || !longitudeParseResult)
+        {
+            return new OperationResult(false, "The specified lat/long values are not valid decimals.");
+        }
```

Comment:
Can't we create an `IValidator<>` for the request these came from and validate them before they hit the function? We should be able to do this check higher up and pass decimals rather than strings down to the service/store